src/compiler/diagnosticMessages.json

```diff
@@ -951,6 +951,18 @@
         "category": "Error",
         "code": 1338
     },
+    "Module '{0}' does not refer to a value, but is used as a value here.": {
+        "category": "Error",
+        "code": 1339
+    },
+    "Module '{0}' does not refer to a type, but is used as a type here.": {
+        "category": "Error",
+        "code": 1340
+    },
+    "Import specifier must be a string literal type, but here is '{0}'.": {

```

Comment:
nit, not sure the type name here gives you much value.. we only allow string literals, not even strings.. so i would just say only string literals are valid import specifiers.

Reply:
We'll parse any type there to be permissive, though; this is more of a grammar error.

Reply:
I know. i am just saying whatever `{0}` is, the error message is just saying you cannot have it.. in other words, the type name is extraneous

Reply:
I find including the type in the error message useful as frequently an error only exists because I believe that the type is correct at a given location; so seeing where reality differs from my expectation is useful.